src/Altinn.AccessManagement.Core/Helpers/PolicyHelper.cs

```diff
@@ -695,6 +801,34 @@ public static List<List<PolicyAttributeMatch>> GetRulePolicyAttributeMatchesForC
             return ruleAttributeMatches;
         }
 
+        /// <summary>
+        /// Check of a given app definition exist as a subject in a list of subjects 
+        /// </summary>
+        /// <param name="ruleSubjects">The subject list to check</param>
+        /// <param name="delegaterApp">the app to check for</param>
+        /// <returns>true if the app deffinion exist in list of subjects and false if not</returns>
+        public static bool ContainsDelegatorAppInSubject(List<List<PolicyAttributeMatch>> ruleSubjects, IEnumerable<AttributeMatch> delegaterApp)
```

Comment:
Ikke en fan av å ha egen kode for å gjøre det som ABAC'en er skrevet for å gjøre: matche en request subject mot regler i en policy. 

Så vil helst at vi løser GetDelegableAppRights på samme måte som vi løser det for personer:
- ved å lage en xacml request for org/app subject basert på delegerende app
- splitte alle enkelt regler fra app policy i enkeltregel policies
- kalle ABAC pr. enkeltregel policy for å se på org/app subject matcher enkelt regler fra app policy

Vi vet ikke om tjenesteeier har slått sammen regler for Appen med andre Rolle/Tilgangspakke subject regler f.eks (selv om vår eksempel policy hvertfall nok kommer til å ha disse i egne regler som am-devtest-instancedelegation eksempel appen jeg satte opp)

Reply:
Egen Issue og PR stenger denne her